src/shared/pages/team/editTeam/editTeam.tsx

```diff
@@ -26,13 +37,29 @@ const validationSchema = object({
         .label('new team name')
 });
 
-const EditTeam: React.FC<EditTeamProps> = ({ csrfToken, history, match }) => {
+
+const EditTeam: React.FC<EditTeamProps> = ({ csrfToken, history, match, hasRole }) => {
 
     const [pageError, addFormError, clearErrors, setErrorMessage] = useError('', VALIDATION_ERROR_TITLE);
     const [state, dispatch] = useReducer(reducer, initialState);
 
     const { params: { teamId } } = match;
 
+    const getUnitsForTypeAhead = useCallback(() => new Promise<Item[]>(resolve => getUnits()
```

Comment:
IntelliJ flags this as duplicated code fragment in three locations, potential to break out separately and avoid duplication?

Reply:
Also on reflection is there anything special about this in regards to the TypeAhead, or would a generic `getUnits` suffice?

Reply:
I have moved this out into another file. It's special in that it copies specific unit fields into the label/value fields of the Item.